www/segment-events/constants.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Events are identified by the following structured pattern:
+ * openedx.paragon.event_environment.event_name.event_action
+ */
+
+export const ICON_COPIED = 'openedx.paragon.docs.icons-table.selected-icon.copied';
+export const LEAVE_FEEDBACK_CLICKED = 'openedx.paragon.docs.leave_feedback.clicked';
+export const PLAYGROUND_CLICKED = 'openedx.paragon.docs.menu.playground.visit_playground.clicked';
+export const CONTRAST_CHECKER_CLICKED = 'openedx.paragon.docs.menu.tools.visit_contrast_checker.clicked';
+export const PLAYGROUND_URL_COPIED = 'openedx.paragon.docs.playground.copy-url.copied';
+export const PAGE_EDIT_BTN_CLICKED = 'openedx.paragon.docs.page_edit.clicked';
+export const SHADOW_GENERATOR_LAYER_ENABLED = 'openedx.paragon.docs.elevation.shadow-generator.layer.enabled';
+export const SHADOW_GENERATOR_LAYER_DISABLED = 'openedx.paragon.docs.elevation.shadow-generator.layer.disabled';
+export const SHADOW_GENERATOR_LAYER_REMOVED = 'openedx.paragon.docs.elevation.shadow-generator.layer.removed';
+export const SHADOW_GENERATOR_LAYER_ADDED = 'openedx.paragon.docs.elevation.shadow-generator.layer.added';
+export const SHADOW_GENERATOR_LAYER_UPDATED = 'openedx.paragon.docs.elevation.shadow-generator.layer.updated';
+export const COMPONENT_USAGE_LINK_CLICKED = 'openedx.paragon.docs.usage-insights.component-usage-link.clicked';
+export const INSIGHTS_TAB_CLICKED = 'openedx.paragon.docs.insights.tab.clicked';
+export const SETTINGS_OPENED = 'openedx.paragon.docs.settings.opened';
+export const SETTINGS_CLOSED = 'openedx.paragon.docs.settings.closed';
+export const SETTINGS_CHANGED = 'openedx.paragon.docs.setting.changed';
+export const EXAMPLE_CODE_BLOCK_WITH_HEADING_CLOSED = 'openedx.paragon.docs.ui.example-code-block.closed';
+export const EXAMPLE_CODE_BLOCK_WITH_HEADING_OPENED = 'openedx.paragon.docs.ui.example-code-block.opened';
+export const EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_CLOSED = 'openedx.paragon.docs.ui.example-code-block.without-heading.closed';
+export const EXAMPLE_CODE_BLOCK_WITHOUT_HEADING_OPENED = 'openedx.paragon.docs.ui.example-code-block.without-heading.opened';
```

Comment:
Would you consider to make names smaller by creating groups like:

```
export const ICON_EVENTS = {
  COPIED: 'openedx.paragon.docs.icons-table.selected-icon.copied',
  // ... other icon related events
};

export const SETTINGS_EVENTS = {
  OPENED: 'openedx.paragon.docs.settings.opened',
  CLOSED: 'openedx.paragon.docs.settings.closed',
  CHANGED: 'openedx.paragon.docs.setting.changed',
  // ... other settings related events
};
```

It feels like it need `EVENT` word in names but also it will make them longer. Suggestion will solve both problems altogether. What do you think?

Reply:
ok, done